packages/apps/job-launcher/server/src/modules/job/job.service.ts

```diff
@@ -1467,7 +1470,10 @@ export class JobService {
 
     const kvStoreClient = await KVStoreClient.build(signer);
 
-    const feeValue = await kvStoreClient.get(oracleAddress, KVStoreKeys.fee);
+    const feeValue = await kvStoreClient.get(
+      oracleAddress || signer.address,
```

Comment:
remove signer.address and also `const signer = this.web3Service.getSigner(chainId);`

Reply:
The line const `signer = this.web3Service.getSigner(chainId)` is being used to create the KVStore client